PatientWebApplication/UserMicroserviceApi/Model/PhoneNumber.cs

```diff
@@ -0,0 +1,22 @@
+
+/***********************************************************************
+ * Module:  PhoneNumber.cs
+ * Author:  Tamara
+ * Purpose: Definition of the Class Model.Patient.PhoneNumber
+ ***********************************************************************/
+using System;
+
+namespace UserMicroserviceApi.Model
+{
+    public class PhoneNumber : Entity
+    {
+        public int number { get; set; }
+        public string name { get; set; }
```

Comment:
Change properties names to start with upper letter and apply in rest of code (remove 'this' from constructors).